src/common/requests.js

```diff
@@ -0,0 +1,43 @@
+function request(method, url, headers, body) {
+  return fetch(url, {
+    method,
+    headers: new Headers(headers),
+    body,
+  });
+}
+
+export function get(url, headers) {
```

Comment:
Let`s change it a bit to allow us pass an object with params like
```
{ 
 id:2,
}
```
This params will be send as URL params. Also this change will help us to do less ctrl+c, ctrl+v code in services